src/core/Akka.Remote/UniqueAddress.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using Akka.Actor;
+
+namespace Akka.Remote
+{
+    // ARTERY: Document that UniqueAddress is a copy of the same class from Akka.Cluster.Member
+    // ARTERY: Temporarily use internal for now, so we don't break API approval list
+    /// <summary>
+    /// Member identifier consisting of address and random `uid`.
+    /// The `uid` is needed to be able to distinguish different
+    /// incarnations of a member with same hostname and port.
+    /// </summary>
+    internal class UniqueAddress : IComparable<UniqueAddress>, IEquatable<UniqueAddress>, IComparable
```

Comment:
This code duplication is the same as in the scala code, UniqueAddress code is duplicated in Akka.Cluster and Akka.Remote